### PR TITLE
chore(module:statistic): refactor

### PR DESCRIPTION
**Pull request by wendellhu95**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/NG-ZORRO/ng-zorro-antd/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Application (the showcase website) / infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?
```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: components/statistic/statistic.spec.ts [hunk 1]
```diff
@@ -1,26 +1,23 @@
-import { CommonModule } from '@angular/common';
 import { Component, DebugElement } from '@angular/core';
-import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { ComponentFixture } from '@angular/core/testing';
 import { By } from '@angular/platform-browser';
-import { NzStatisticComponent } from './nz-statistic.component';
-import { NzStatisticModule } from './nz-statistic.module';
+import { ComponentBed, createComponentBed } from 'ng-zorro-antd/core/testing/componet-bed';
```
Comment (vthinkxie) [inline components/statistic/statistic.spec.ts]: as createComponentBed

File: components/core/pipe/nz-time-range.spec.ts [hunk 2]
```diff
@@ -3,7 +3,7 @@ import { Component } from '@angular/core';
 import { ComponentFixture, TestBed } from '@angular/core/testing';
 
 import { NzPipesModule } from 'ng-zorro-antd/core/pipe';
-import { NzStatisticModule } from '../../statistic/nz-statistic.module';
+import { NzStatisticModule } from '../../statistic/statistic.module';
```
Comment (vthinkxie) [inline components/core/pipe/nz-time-range.spec.ts]:
```suggestion
import { NzStatisticModule } from 'ng-zorro-antd/statistic';
```

File: components/statistic/countdown.component.ts [hunk 3]
```diff
@@ -23,15 +23,27 @@ import {
...
+      [nzPrefix]="nzPrefix"
+      [nzSuffix]="nzSuffix"
+    >
+    </nz-statistic>
+
+    <ng-template #countDownTpl>{{ diff | nzTimeRange: nzFormat }}</ng-template>
+  `
 })
 export class NzCountdownComponent extends NzStatisticComponent implements OnInit, OnChanges, OnDestroy {
   /** @override */
```
Comment (vthinkxie) [inline components/statistic/countdown.component.ts]: delete /** @override */

File: components/statistic/countdown.component.ts [hunk 4]
```diff
@@ -23,15 +23,27 @@ import {
 } from '@angular/core';
 import { interval, Subscription } from 'rxjs';
 
-import { REFRESH_INTERVAL } from './nz-statistic-definitions';
-import { NzStatisticComponent } from './nz-statistic.component';
+import { NzStatisticComponent } from './statistic.component';
+import { REFRESH_INTERVAL } from './typings';
```
Comment (vthinkxie) [inline components/statistic/countdown.component.ts]: should not inside the typings file

File: components/statistic/statistic.spec.ts (hunk 1 above)
Comment (wendellhu95) [reply components/statistic/statistic.spec.ts]: 这里加上 θ 然后再 as 回来，出于什么样的考虑？

Comment (vthinkxie) [reply components/statistic/statistic.spec.ts]: 避免对外暴露，禁止外部用户使用，与angular同样用法

Comment (vthinkxie) [reply components/statistic/statistic.spec.ts]: 不过内部直接二级引入好像也问题不大

**Pull request merged**